packages/core/src/process/1-prepare-executor.ts

```diff
@@ -36,6 +38,7 @@ export class PrepareExecutor {
       .provideClass(coreTokens.optionsValidator, OptionsValidator);
     const configReader = configReaderInjector.injectClass(ConfigReader);
     const options: StrykerOptions = await configReader.readConfig(cliOptions);
+    const preDefinedLevel: PartialStrykerOptions = await configReader.readJsonConfig(path.resolve('../packages/core/src/predefinedMLevels.json'));
```

Comment:
I think it would be better to move this constant to a separate file, which will export the variable.